pkg/sql/txn_restart_test.go

```diff
@@ -1680,7 +1680,7 @@ CREATE TABLE t.test (k INT PRIMARY KEY);
 	// moveOutOfAutoRetry dictates whether we move the transaction from state
 	// AutoRetry to Open before starting the batch of statements that will
 	// encounter the push. When moveOutOfAutoRetry is set, we expect to get a
-	// retriable error at commit time. When it's not, we expect to see no error
+	// retriable error. When it's not, we expect to see no error
```

Comment:
nit: please re-wrap comments at 80 columns

Reply:
@andreimatei, sorry, I don't understand what do you mean here?